packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -107,7 +107,10 @@
     }
 
     div.paragraph {
-        .legacy-typography;
+        font-family: "Noto Serif", serif;
```

Comment:
**suggestion**: It _might_ be worth creating a variable for this font-family, but I don't feel super strongly about it.